.eslintrc

```diff
@@ -12,6 +12,9 @@
   "rules": {
     "no-console": "off",
     "strict": ["error", "global"],
-    "curly": "warn"
+    "curly": "warn",
+    "getter-return": "error",
+    "function-paren-newline": ["error", "multiline"],
```

Comment:
As far as I can see this is a style rule. We don't enable any style rules in this preset.